server/DEVELOPER.MD

```diff
@@ -0,0 +1,88 @@
+# DEVELOPER wiki
+
+## Building the app
+
+This document describes how to build the entire app in a simulated local production environment (staging) using docker-compose.
+
+[Prerequisite Software](#prerequisite-software)
+
+### Prerquisite Software
+
+- [free version of docker desktop](https://www.docker.com/get-started)
+
+## Build
+
+- cd to `server`
+
+- Delete development DB: from the `server` directory remove the `pgdata` directory if it exists
+
+```sh
+# see if it exists
+ls
+
+# delete if it exists
+rm -rf pgdata
+```
+
+- in the `.env` file, change the host to `db` (the docker-compose service name) and the port to `5432`:
+
+```.env
+# if you change DATABASE_DB, also change db name in init.sql
+PORT=3001
+DATABASE_HOST=db
+DATABASE_PORT=5432
```

Comment:
@jd2rogers2 , Thanks for reviewing. I modified the docker-compose file to use a different base on another port to avoid this problem. It's documented in the `DOCKER_BUILD.md` file. Let me know if it works for you.